templates/src/csharp/Grpc.Core/Internal/NativeMethods.Generated.cs.template

```diff
@@ -0,0 +1,233 @@
+%YAML 1.2
+--- |
+  <%
+    native_method_signatures = [
+      'void grpcsharp_init()',
```

Comment:
Yes, that would be nice but would require some extra logic that could end up being quite complex (as the mapping of signatures in not always straightforward). 
One day I might add this, but for now leaving as is because this change is not removing any preexisting check.